README.md

```diff
@@ -171,6 +171,7 @@ Some [Groupware](#groupware) solutions also feature calendar/address book editin
   * [CalendarServer](http://trac.calendarserver.org/) - A standards-compliant server implementing the CalDAV and CardDAV protocols. `Apache` `Python`
   * [calypso](http://keithp.com/calypso/) - A python-based CalDAV/CardDAV server. ([Source Code](http://keithp.com/cgi-bin/gitweb.cgi?p=calypso.git;a=tree)) `GPLv3` `Python`
   * [DAViCal](http://www.davical.org/) - A server for calendar sharing (CalDAV). ([Source Code](https://gitlab.com/davical-project/davical)) `MIT` `PHP`
+   * [Kcaldav](https://kristaps.bsd.lv/kcaldav/) - is a simple, safe, and minimal CalDAV server depending only on [kcgi(3)](http://kristaps.bsd.lv/kcgi/), [SQLite](https://www.sqlite.org/), and [expat(3)](http://expat.sourceforge.net/). ([Source code](https://github.com/kristapsdz/kcaldav))  `ISC`   `C`
```

Comment:
There should be only 1 space between the source code and the license and only 1 between the license and the language.

Is this a fork of DAVical? If not remove one space before the *.